### PR TITLE
fully qualify dockerhub imagestream sources

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

File: examples/db-templates/mariadb-ephemeral-template.json [hunk 1]
```diff
@@ -5,16 +5,16 @@
...
+      "description": "MariaDB database service, without persistent storage. For more information about using this template, including OpenShift considerations, see https://github.com/sclorg/mariadb-container/blob/master/10.2/root/usr/share/container-scripts/mysql/README.md.\n\nWARNING: Any data stored will be lost upon pod destruction. Only use this template for testing",
       "iconClass": "icon-mariadb",
       "tags": "database,mariadb",
       "openshift.io/long-description": "This template provides a standalone MariaDB server with a database created.  The database is not stored on persistent storage, so any restart of the service will result in all data being lost.  The database name, username, and password are chosen via parameters when provisioning this service.",
       "openshift.io/provider-display-name": "Red Hat, Inc.",
-      "openshift.io/documentation-url": "https://github.com/sclorg/mariadb-container/blob/master/10.1/README.md",
+      "openshift.io/documentation-url": "https://github.com/sclorg/mariadb-container/blob/master/10.2/root/usr/share/container-scripts/mysql/README.md",
```
Comment (jim-minter) [inline examples/db-templates/mariadb-ephemeral-template.json]: I wonder whether just `https://github.com/sclorg/mariadb-container/` might be more resistant to subsequent repo reorgs?

File: examples/quickstarts/cakephp-mysql-persistent.json [hunk 2]
```diff
@@ -17,8 +17,8 @@
   },
   "message": "The following service(s) have been created in your project: ${NAME}, ${DATABASE_SERVICE_NAME}.\n\nFor more information about using this template, including OpenShift considerations, see https://github.com/openshift/cake-ex/blob/master/README.md.",
   "labels": {
-    "template": "cakephp-mysql-persistent",
-    "app": "cakephp-mysql-persistent"
+      "template": "cakephp-mysql-persistent",
```
Comment (jim-minter) [inline examples/quickstarts/cakephp-mysql-persistent.json]: these whitespace changes don't appear to be visible upstream?

File: examples/db-templates/mariadb-ephemeral-template.json (hunk 1 above)
Comment (bparees) [reply examples/db-templates/mariadb-ephemeral-template.json]: that commit came from the SCL team, they own it now so it's up to them how/where they want it to point.

File: examples/quickstarts/cakephp-mysql-persistent.json (hunk 2 above)
Comment (bparees) [reply examples/quickstarts/cakephp-mysql-persistent.json]: also came from SCL, I may have missed that they didn't run the import script on the PR but it'll get resolved next time around I suppose.

**Comment by bparees** [comment]
> That was a particularly parsimonious PR title given all the changes in there ;) Looks fine to me bar the 2 questions.

I was building on top of another PR.  This PR is actually only the second (now only) commit.

**Comment by bparees** [comment]
(I also wish the bot or github or whatever would not automatically start begging for reviews, I was just going to wait until the other PR had merged to rebase this and then solicit a review to avoid confusion)

**Comment by bparees** [comment]
/hold
(want to make sure this doesn't break imageeco tests)

**Comment by soltysh** [comment]
> (want to make sure this doesn't break imageeco tests)

It should not, it's just a different pull spec but in majority of cases it should just work. If it does, it'd be worth fixing those places, since this means they contain weird errors that might bite us in the future.

**Comment by bparees** [comment]
> It should not, it's just a different pull spec but in majority of cases it should just work. 

sure, it's just a sanity check.

**Comment by bparees** [comment]
no new/surprising image eco failures

/hold cancel

**Pull request merged**